sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/Utility.java

```diff
@@ -241,4 +244,27 @@ public static OffsetDateTime parseDate(String dateString) {
                 }
             });
     }
+
+    /**
+     * Reads data from an input stream and writes it to an output stream.
+     * @param source {@link InputStream source}
+     * @param writeLength The length of data to write.
+     * @param destination {@link OutputStream destination}
+     * @throws IOException If an I/O error occurs.
+     */
+    public static void copyToOutputStream(InputStream source, long writeLength, OutputStream destination)
```

Comment:
Could this go into `StorageImplUtils` for the time being? Do we expected or would want a customer to use this API in the near-term?

Reply:
I have nothing against doing that

Reply:
done